test/test_constraint.cpp

```diff
@@ -25,8 +23,6 @@ BOOST_AUTO_TEST_CASE( mock_constraint_is_supported_by_compilers_with_variadic_ma
     BOOST_CHECK( constraint_2( 0, 0 ).c_( 0 ) );
 }
 
-#endif // MOCK_VARIADIC_MACROS
-
 namespace
 {
     MOCK_CONSTRAINT_EXT( constraint_0_ext, 0,, actual == 0 )
```

Comment:
Do we still need `MOCK_CONSTRAINT_EXT`?

Reply:
You are right, we don't. It is an implementation detail now